homework-no-git/06_14_2018-homework/js/app.js

```diff
@@ -0,0 +1,9 @@
+// The difference between a parameter and an argument is that parameter is a discription of what needs to be there and and argument is what is actually being calculated.
+//a console.log you are unable to return and use the result
```

Comment:
awesome — yep — console.log() just for printing stuff — a debugging tool for developers